apps/rbac/nonprod-role.yaml

```diff
@@ -17,7 +17,7 @@ rules:
   verbs: ['create']
 - apiGroups: ['']
   resources: ['services']
-  verbs: ['delete']
+  verbs: ['create', 'delete']
```

Comment:
why do u need create for ? delete is given on non-prod only to manage failed releases